Documentation/Signoffs/SensorSignoff.md

```diff
@@ -0,0 +1,198 @@
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Within this Sensor Subsystem, the design team has decided to get multiple types of sensors for the modularity of the device. These types are digital, analog voltage, and analog current sensors. While the digital sensors send bits describing the measurement of a specific parameter, the analog sensors send their signals a bit differently. Analog sensors output analog signals which are different to measure from bits, therefore the Interface Subsystem will provide assistance in parameter conversion to the microcontroller.
+
+## CONSTRAINTS ON THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Constraints on the subsystem are as follows:
+
+1. The pollutants of the Air Quality Device shall be able to detect carbon dioxide, methane, nitrogen dioxide, ozone, particulate matter, and sulfur dioxide. These sensors should work independently from each other.
+
+2. Shall be modular. Specifically, the change of sensors should be seamless and handled by the board and software. The sensors will have a variety of Input/Output (I/O) ports. For a sensor to be in the same group, it must share the same voltage supply, current consumption, and communication technique through the interface. The sensor shall meet either of the following output ranges [9]:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; ● Output the signal at the range of 0 - 10 VDC for analog voltage.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; ● Output 4 - 20 mA for analog current/digital.
+
+3. Each sensor must be able to communicate with the microcontroller using a communicative language. The sensors shall communicate through Serial Peripheral Interface (SPI), Universal Asynchronous Receiver-Transmitter (UART), or Inter-Integrated Circuit (I2C). The microcontroller for this design will also be able to communicate with those same data transfer techniques. 
+
+4. For windy environments, the sampling rate should be at most 10 Hz. This is because there are environmental factors that can cause a loss of accuracy in the data given higher wind speeds or non preferable weather conditions. After viewing this experiment of wind analysis in Thailand, it can be shown that using the Nyquist sampling theorem and varied wind speed, one can find the appropriate sampling rate at higher wind speed conditions without losing as much accuracy. They observed the wind speed data and used Fourier Transform to locate a cutoff frequency for the Nyquist sample rate. From their calculations, they have also concluded 10 Hz to be a suitable sample rate for higher wind conditions [1]. For indoor monitoring, ambient gas sensors will be sampled at the lowest rate of 2 Hz per second based on the diffusion of ambient air quality [3].
+
+5. Also because of noise, resolution for each analog sensor must be at most 0.15 ppm or less. For digital sensors, a max resolution of 0.1 ppm or less shall be used for a 1 Hz sample rate.
```

Comment:
Resolution needs to be specified per item measured as each has a different range. For example the NO2 range listed below is from 53 ppb to 100 ppb so a 0.1 ppm resolution is not appropriate. Also don't intermix sample rate and resolution - they are separate items. It is stated "Because of noise..." but it is unclear how noise determines the resolution.

Reply:
Ok, I will remove number 5. Plan to remove the sample rate mentioned in number 5 (because of the 10-2 Hz range), and move both resolution values to 8.